Contest_API.md

```diff
@@ -1549,6 +1549,10 @@ For some common award cases the following IDs should be used.
 
 Clients with the `admin` role may make changes to awards using the normal [HTTP methods](#http-methods) as specified above. Specifically, they can POST new awards, PATCH one or more attributes, or DELETE an existing award. All requests must include the id attribute.
 
+Each award id may be managed by the server, overridable, or unmanaged (potentially even unknown). Managed awards are assigned by the server and any attempt to change them will fail. Overridable awards will be assigned by the server by default, but once it has been modified externally the server will stop assigning or updating them. Award ids that are unmanaged or unknown to the server are solely up to the client.
+
+For instance, a server may be configured to assign `winner` and `*-medal` awards as managed awards, and these cannot be overridden by clients. The same server may assign `first-to-solve-*` awards by default, but allow a client to override them. This server may have no support for assigning `group-winner-*` and these will only exist if a client adds them. The client can also add other or arbitrary awards like `first-submission-for-country`.
```

Comment:
As written the first sentence seems to claim that it is a property of `winner` and `*-medal` that these could not be overridden by clients. But this is just an example, right? I would suggest changing "and these cannot be" to "and to not allow these to be". That said, this paragraph might change/disappear based on my earlier comment?